### DISCUSSION TITLE
Hidden YAML keyword:  wait

### DISCUSSION THREAD

**Discussion by Beliefg** [artilleryio/artillery#2764] [Artillery]
I've discovered this keyword wait, that seems to be the only wait to get:
websocket.messages_sent: ....................................................... 2
websocket.send_rate: ........................................................... 1/sec

but it does not appear documented anywhere. Can someone provide insight into it?  To provide greater context, I came across while trying to listen to an open socket that is sending messages triggered elsewhere.  Here is the scenario: 

  scenarios:
    - name: Capture WS message
      engine: ws # Enable the Websocket engine
      flow:
        - connect: '{{ target }}/{{ userKey }}&sessionId={{ userSessionId }}&officeKey={{ officeKey }}'
        - log: 'Connected to WebSocket'
        - think: 1
        - log: "Waiting for message..."
        - wait: 10  # Wait for up to 10 seconds for the message

**Comment by AnshumanRohella** [comment]
I saw it in the tests as well. I am stress testing my websocket as well and I want to test the responses on certain socket events to have some metrics around actual throughput.
